cpp/client.cc

```diff
@@ -366,13 +371,35 @@ int main(int argc, char** argv) {
         return 1;
       }
     }
+
+    start_pos = arg_val.find(arg_str_affinity);
+    if (start_pos != std::string::npos) {
+      start_pos += arg_str_affinity.size();
+      if (arg_val[start_pos] == '=') {
+        if (arg_val.substr(start_pos + 1) == "true") {
```

Comment:
Nit: Do we need a value with --affinity flag since it is just true or false? Just the presence of the flag means affinity is desired.

Reply:
all our boolean arguments in all languages have this same behaviour: needing true or false and rejecting the syntax "--arg"